dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -241,7 +243,7 @@ public String buildInsertCommand(String tableName,
     }
 
     public DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors) {
```

Comment:
Now I'm not so sure. I've now made the new interface `DbStoredProcedureCall` somewhat redundant :( I'm in danger of slowly moving into the area of #463 in this PR. I can see the evolution to `DbCommand` / `DbStatement` again now.

I can roll this back. Are we happy with the current `DbStoredProcedureCall` and the related classes for this PR?